### PR TITLE
P20 1001/confirmation banner after account linking

### PR DESCRIPTION
**Pull request by alex-m-brown**
Adds a confirmation banner after linking an existing account to an LMS account. This banner will only appear when the LMS target_link_uri is set to certain pages. There are more details in the [Jira ticket](https://codedotorg.atlassian.net/browse/P20-1001). 

image

## Links

- Jira Ticket: [P20-1001](https://codedotorg.atlassian.net/browse/P20-1001)

### CODE REVIEW

File: dashboard/app/controllers/lti/v1/account_linking_controller.rb [hunk 1]
```diff
@@ -50,6 +50,7 @@ def link_email
             metadata: metadata,
           )
           target_url = session[:user_return_to] || home_path
+          flash[:notice] = I18n.t('lti.account_linking.successfully_linked')
```
Comment (stephenliang) [inline dashboard/app/controllers/lti/v1/account_linking_controller.rb:53]: Could we add a test in both controllers to ensure the flash is displayed?

File: dashboard/test/controllers/lti/v1/account_linking_controller_test.rb [hunk 2]
```diff
@@ -37,6 +37,7 @@ class Lti::V1::AccountLinkingControllerTest < ActionController::TestCase
       )
     )
     post :link_email, params: {email: @user.email, password: 'password'}
+    assert_equal "You have successfully linked your Code.org account!", flash[:notice]
```
Comment (stephenliang) [inline dashboard/test/controllers/lti/v1/account_linking_controller_test.rb]:
```suggestion
    assert_equal I18n.t('lti.account_linking.successfully_linked'), flash[:notice]
```

File: dashboard/test/controllers/omniauth_callbacks_controller_test.rb [hunk 3]
```diff
@@ -1798,6 +1798,7 @@ class OmniauthCallbacksControllerTest < ActionController::TestCase
           # so this includes 1 email, 1 SSO, and 1 LTI auth option
           _(user.authentication_options.count).must_equal 3
           _(user.authentication_options).must_include lti_auth_option
+          assert_equal "You have successfully linked your Code.org account!", flash[:notice]
```
Comment (stephenliang) [inline dashboard/test/controllers/omniauth_callbacks_controller_test.rb]:
```suggestion
          assert_equal I18n.t('lti.account_linking.successfully_linked'), flash[:notice]
```

**Review by nicklathe — APPROVED** [review]
Great work!

**Pull request merged**